File: Team_Workspace/dalia_ahmed/task5.py

volts=[12.5, 11.8, 13.1, 12.0]
def find_extremes(list):
    max=list[0]
    min=list[0]
    for i in list:
        if i>max:
         max=i
        if i<min:
           min=i
    return max,min
max,min=find_extremes(volts)
print("max value",max)
print("min value",min)

File: Team_Workspace/Fatma_Ahmed/cpp_tasks/task5/task5.cpp
// task5.
#include <iostream>
#include <vector>

using namespace std;

void find_extremes(const vector<double>& list,float &max,float &min) {
	max = min = list[0];

	for (float num : list) {
		if (num > max)
			max = num;
		if (num < min)
			min = num;

	}

}

int main()
{
	std::vector<double> volts = { 12.5, 11.8, 13.1, 12.0 };

	float min_v=0.0, max_v = 0.0;
	find_extremes(volts, max_v, min_v);

	cout << "max =  " << max_v << endl;
	cout << "min = " << min_v << endl;

}

